Makefile

```diff
@@ -1,4 +1,5 @@
-#
+PACKAGES:=$(shell go list ./... | grep -v /vendor/)
```

Comment:
I believe you can also write:
```
TARGET1 TARGET2: PACKAGES = $(shell go list .....)
```
Or if you have more specific sets of paths for for a target than `./...` you can use that.